CONTRIBUTING.md

```diff
@@ -167,7 +167,7 @@ tox run -e unit,coverage-report
 
 ### Integration Testing
 
-This repo uses `pytest` and `pytest-operator` to execute functional/integration tests against the charm files. The integration tests are defined in `./tests/integration`. The charms are deployed according to the bundle defined in `./tests/integration/test-bundle.yaml`.
+This repo uses `pytest-jubilant` and `Jubilant` to execute functional/integration tests against the charm files. The integration tests are defined in `./tests/integration`. The charms are deployed according to the bundle defined in `./tests/integration/test-bundle.yaml`.
```

Comment:
Note that there are several other mentions of `pytest-operator` still in [CONTRIBUTING.md](https://github.com/canonical/infra-backup-operator/blob/main/CONTRIBUTING.md). This won't block public listing, of course, but it would be good for you to go over the whole thing.